gen/_template/json/stdmarshaler.tmpl

```diff
@@ -0,0 +1,25 @@
+{{- define "json/stdmarshaler" }}
+{{- /*gotype: github.com/ogen-go/ogen/internal/ir.Type*/ -}}
+{{- $g := $.GenericOf -}}
+// MarshalJSON implements stdjson.Marshaler.
+func (s {{ $.Name }}) MarshalJSON() ([]byte, error) {
+	e := jx.Encoder{}
+	{{- if $.IsGeneric }}
+	s.Encode(&e{{ if $g.Format }}, json.Encode{{ $g.JSON.Format }}{{ end }})
+	{{- else }}
+	s.Encode(&e)
+	{{- end }}
+	return e.Bytes(), nil
+}
+
+// UnmarshalJSON implements stdjson.Unmarshaler.
+func (s {{ $.Name }}) UnmarshalJSON(data []byte) error {
```

Comment:
Probably there should be pointer receiver

Reply:
Probably.